pkg/snmp/traps/config.go

```diff
@@ -37,6 +38,8 @@ type Config struct {
 	CommunityStrings      []string `mapstructure:"community_strings" yaml:"community_strings"`
 	BindHost              string   `mapstructure:"bind_host" yaml:"bind_host"`
 	StopTimeout           int      `mapstructure:"stop_timeout" yaml:"stop_timeout"`
+	AuthoritativeEngineID [28]byte `mapstructure:"-" yaml:"-"`
```

Comment:
Is this intentional addition? There is already a field named like this, and capitalized version is not used in this PR.